netlify/functions/metrics.ts

```diff
@@ -31,12 +31,9 @@ export const handler: Handler = async (event: HandlerEvent) => {
       ...identifySourceAndEvent(event.headers),
     };
 
-    const result: unknown = await collectMetricsHandler(rawEvent);
+    await collectMetricsHandler(rawEvent);
 
-    return {
-      statusCode: 200,
-      body: JSON.stringify(result),
-    };
+    return { statusCode: 201 };
```

Comment:
Is it not only a 201 when we collected and wrote metric data?
In case no metric condition matches we don't write data, then a 200 would be more appropriate.

We could got with a 204, which means that the server processed the request but we do not return any content.

Reply:
I don't think we should decide based on what happened - the caller shouldn't find out if we saved something or not. I was just looking for an OK without content, I guess 204 works too/better for that.